packages/governance/test/unitTests/test-buildParamManager.js

```diff
@@ -320,6 +320,41 @@ test('Strings', async t => {
   });
 });
 
+test('JSON blobs', async t => {
+  const paramManager = makeParamManagerBuilder()
+    .addNat('Acres', 50n)
+    .addJson('Blob', '{ "author": "Crockford" }')
+    .build();
+  t.is(paramManager.getJson('Blob'), '{ "author": "Crockford" }');
```

Comment:
shouldn't the result be the JS value instead of the string?
```suggestion
  t.is(paramManager.getJson('Blob'), { "author": "Crockford" });
```

Requesting that change or some documentation on why the `string` type is not sufficient.

Reply:
It's a good question. My strong suspicion is that for governance purposes, we're talking about json values encoded as strings. I think people want to see the stringified version when they're voting, and I think governed values that are specified as json would be desired by the contract as json strings.

But I could be wrong about that. Particularly since the contract that I thought was using json actually uses the value of its parameter as a record. (I'm adding that in a separate PR.)

What do you think? Would it make sense to specify it as a string and extract the value as an object?

Reply:
>  I think people want to see the stringified version when they're voting

Not sure about this. I think they want to see the data. If it were a tree that'd probably be easier to read. 

> and I think governed values that are specified as json would be desired by the contract as json strings.
> But I could be wrong about that. Particularly since the contract that I thought was using json actually uses the value of its parameter as a record. (I'm adding that in a separate PR.)

Yeah, I think if it's JSON the consumer would want the data itself.

> Would it make sense to specify it as a string and extract the value as an object?

👍 to extracting the value as an object. Not sure about the _string_ encoding, in light of @erights question about supporting any passable.